pkg/services/queryhistory/database.go

```diff
@@ -265,3 +265,56 @@ func (s QueryHistoryService) unstarQuery(ctx context.Context, user *models.Signe
 
 	return dto, nil
 }
+
+func (s QueryHistoryService) migrateQueries(ctx context.Context, user *models.SignedInUser, cmd MigrateQueriesToQueryHistoryCommand) error {
+	err := s.SQLStore.WithTransactionalDbSession(ctx, func(session *sqlstore.DBSession) error {
+		var instertError error
```

Comment:
```suggestion
		var insertError error
```

Reply:
Nice catch. You have better 👀 than my spellcheck plugin! 🤩